### ISSUE TITLE
[Accesibility] DetailsRow should show LocalizedControlType as "DataItem" instead of "Row" in inspect tool

### ISSUE THREAD

**Issue by akashperfect** [OfficeDev/office-ui-fabric-react#3246]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question)

Are you willing to submit a PR to fix? 

Requested priority: High

Products/sites affected: Internal

#### Describe the issue:

With inspect tool on, 
- navigate to DetailsRow inside a DetailsList
- LocalizedControlProperty is defined as "Row" 

LocalizedControlProperty  (LCT) should be defined as Data item as the list is defined as Data grid and as per the below reference link, there is no such LCT called "Row"
https://msdn.microsoft.com/en-us/library/windows/desktop/ee671193(v=vs.85).aspx

![untitled](https://user-images.githubusercontent.com/2953096/32089893-3f32579c-bb0a-11e7-98bb-b5ec36f09a88.png)

#### Actual behavior:

LocalizedControlProperty is defined as "Row"

#### Expected behavior:

LocalizedControlProperty should be "DataItem"

**Comment by KevinTCoughlin** [comment]
I'm unsure of how to properly address since the [ARIA documentation][0] states:

>Authors MUST ensure that elements with role gridcell are owned by elements with role **row**, which in turn are owned by an element with role rowgroup, **grid** or treegrid. If the author applies any non-global WAI-ARIA states or properties to a native markup element that is acting as a row (such as the tr element in HTML), the author MUST also apply the role of row, as stated in the section on Implementation in Host Languages. Authors MAY make cells focusable. Authors MAY provide row and column headers for grids, by using rowheader and columnheader roles.

Is this still incorrect? I'd like to fix, but have limited experience with LCT and do not want to introduce an incorrect ARIA role by fixing.

[0]: https://www.w3.org/WAI/PF/aria-1.1/roles#grid

**Comment by KevinTCoughlin** [comment]
Pinging @akashperfect once more because I'm less familiar with LCT. Given the ARIA spec the current attributes seem correct. Would like to know more and how to fix if that's not the case.

**Comment by KevinTCoughlin** [comment]
Closing since the current implementation follows ARIA spec as mentioned above. The issue seems to reside on the tool's side. Please re-open if you believe that to be incorrect with a bit more info so that we can re-assess and fix properly. Thank you!
